src/plugins/mer/mervirtualmachinesettingswidget.cpp

```diff
@@ -112,22 +120,20 @@ void MerVirtualMachineSettingsWidget::initGui()
     });
 
     ui->memoryInfoLabel->setPixmap(Utils::Icons::INFO.pixmap());
-    ui->memoryInfoLabel->setToolTip(
-            QLatin1String("<font color=\"red\">")
-            + tr("Stop the virtual machine to unlock this field for editing.")
-            + QLatin1String("</font>"));
+    setToolTip(ui->memoryInfoLabel, tr("Stop the virtual machine to unlock this field for editing."));
```

Comment:
Nice, we could also use a variable and get rid of the repeated tr(message) finally.